style.css

```diff
@@ -327,14 +327,14 @@ ul li ul {
     padding: 0 0 0 1.5em;
 }
 dt {
-    width: 140px;
```

Comment:
Yeah, I know, but we need a completely different style for the docs anyway - a side-by-side layout doesn't look well when the "heading" is too long.